graphql-dgs-codegen-shared-core/src/main/kotlin/com/netflix/graphql/dgs/client/codegen/GraphQLQuery.kt

```diff
@@ -27,6 +27,7 @@ abstract class GraphQLQuery(val operation: String, val name: String?) {
     constructor() : this("query")
 
     open fun getOperationType(): String? {
+        VariableDefinition.newVariableDefinition()
```

Comment:
Accidentally included in the PR? I don't think it has any effect.

Reply:
yeah, I'm not sure where that came from, will remove.